plugins/push/frontend/public/javascripts/countly.views.component.common.js

```diff
@@ -152,7 +152,7 @@
     countlyPushNotificationComponent.ReviewSectionRow = countlyVue.views.create({
         props: {
             value: {
-                type: String,
+                type: String | Number,
```

Comment:
The correct way of doing the intended thing is [String, Number], as the current expression evaluates to 0.

Reply:
https://stackoverflow.com/a/53608245

Reply:
I saw your note earlier, yet I had no idea I was doing it. I will use arrays.